bolt/test/binary-analysis/AArch64/gs-pacret-multi-bb.s

```diff
@@ -0,0 +1,46 @@
+// RUN: %clang %cflags -march=armv8.3-a -mbranch-protection=pac-ret %s %p/../../Inputs/asm_main.c -o %t.exe
+// RUN: llvm-bolt-binary-analysis --scanners=pacret %t.exe 2>&1 | FileCheck %s
+
+
+// Verify that we can also detect gadgets across basic blocks
```

Comment:
I think the implementation attempts to correctly handle cases like this:

```
    cbnz x0, 1f
    autibsp
    b 2f
1:  autiasp
2:  ret
```

That is, two (or more) basic blocks that all authenticate.

Reply:
Thanks, this is now added in commit a05ae62